target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6018-mr7500.dts

```diff
@@ -0,0 +1,517 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+// Copyright 2024 Weikai Kong (priv at pppig236.com)
+
+/dts-v1/;
+
+#include "ipq6018-512m.dtsi"
+#include "ipq6018-cp-cpu.dtsi"
+#include "ipq6018-ess.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "Linksys MR7500";
+	compatible = "linksys,mr7500", "qcom,ipq6018";
+	
+	aliases {
+		serial0 = &blsp1_uart3;
+		serial1 = &blsp1_uart2;
+		led-boot = &led_system_blue;
+		led-running = &led_system_blue;
+		led-failsafe = &led_system_red;
+		led-upgrade = &led_system_green;
+		ethernet0 = &dp1;
+		ethernet1 = &dp2;
+		ethernet2 = &dp3;
+		ethernet3 = &dp4;
+		ethernet4 = &dp5_syn;
```

Comment:
If you later need to fixup these then there is no point in even having the aliases